tests/integration-tests/tests/createami/test_createami.py

```diff
@@ -141,6 +142,7 @@ def test_build_image(
     )
 
     _test_build_image_success(image)
+    _test_build_instances_tags(image, image.config["Build"]["Tags"], region)
```

Comment:
I think this will be a noop, since the previous `_test_build_image_success` waits for BUILD_COMPLETE, there are no more instances at this execution point

Reply:
unless the describe_instances is also returning the just terminated instances